en/components/themes/examples.md

```diff
@@ -211,12 +211,94 @@ And the result is:
 
 Import the *utilities*, component mixins and the theme functions, define the colors, define the themes and apply them. These are the steps for styling your app with **Ignite UI for Angular Theming**.
 
+
+###Schema Setting
+
+Creating schemas is defining component properties that a theme should have. They are the 'recipes' component themes use when generating theme maps.
+
+To get started, first you have to import the default global schemas ([light-schema]({environment:sassApiUrl}/index.html#variable-light-schema) and [dark-schema]({environment:sassApiUrl}/index.html#variable-dark-schema)):
+
+```scss
+...
+@import '~igniteui-angular/lib/core/styles/themes/utilities';
+...
+```
+Every component has its own light and dark schema, so if you want to make a custom one, you have to extend one of the components default schemas:
+```scss
+//Define the `recipe` for the igx-grid
+$_black-grid-schema: extend($_dark-grid,
+        (
+            content-background:(
+                igx-color:("primary", 100)
+            ),
+            header-background:(
+                igx-color:("secondary", 700)
+            ),
+            cell-selected-background:(
+                igx-color:("primary", 500)
+            ),
+            cell-selected-text-color:(
+                igx-color:("secondary", 500)
+            ),
+            row-hover-background:(
+                igx-color:("secondary", 100)
+            ),
+            header-border-color:(
+                igx-color:("primary", 600)
+            ),
+            content-text-color: white,
+            row-hover-text-color:(
+                igx-color:("primary", 700)
+            ),   
+        )
+);
+//Define the `recipe` for the igx-grid-paginator
+$_black-grid-paginator-schema: extend($_dark-grid-pagination,
+        (
+            text-color:(
+                igx-color: ("secondary", 700)
+            ),
+            background-color:(
+                igx-color: ("primary", 700)
+            ),
+            border-color:(
+                igx-color:( "secondary", 500)
+            )
+        )
+);
+```
+The component schemas are applied through the global schemas, in order to use the custom one you have to extend one of the globals (`$dark-schema` or `$light-schema`):
+```scss
+    $my-black-schema: extend( $dark-schema, (
+            ...
+            igx-grid: $_black-grid-schema,
```

Comment:
Can you also show where how you define the palette, it's quite important, especially with dark schemas.

Reply:
The palette definition within the schemas section is added